### ISSUE TITLE
[BUG] Mod crash when taking a screenshot

### ISSUE THREAD

**Issue by Anonyku05** [KrLite/Flash#7]
### Mod Loader

Fabric

### Minecraft Version

1.20.4

### What Happened?

When taking a screenshot, the flash appear but then the game instantly freezes and crash

### What Should Happen?

Should not freeze and then crash

### Steps to Reproduce

1. Take a screenshot with the mod
2. That's it

### Crash Log

https://mclo.gs/yIxSN80

**Comment by KrLite** [comment]
According to your log, this has nothing to do with **Flash.** It's caused by **Tinted Campfires,** please submit issues to them. I will close this issue as it's not related.

**Comment by Anonyku05** [comment]
Late response but the tinted campfiremod is not installed in my minecraft, i tried using it but it was causing crash so i removed it

**Comment by KrLite** [comment]
> Late response but the tinted campfiremod is not installed in my minecraft, i tried using it but it was causing crash so i removed it

Did you send the wrong log? In your log there's only a crash caused by Tinted Campfires.

**Comment by Anonyku05** [comment]
Yes i think i did, here's the right one: https://mclo.gs/iaKCpJg

**Comment by KrLite** [comment]
> Yes i think i did, here's the right one: https://mclo.gs/iaKCpJg

That is a related crash, however I am very confused by the report - it seems to be a very complicated circumstance that I have never met. I'll check this later.

**Comment by Anonyku05** [comment]
Additionnal info: The crash does not occure when using Linux (Debian 12)

**Comment by Anonyku05** [comment]
Edit: It also crashes on Linux, don't know if the issue will ever be fixed but just wanted to make it know
